### ISSUE TITLE
Search all games with or without Steam Cloud regardless of what the store page says

### ISSUE THREAD

**Issue by squeak9** [SteamDatabase/SteamDatabase#807]
### Feature Description

You can search based on the features like Steam Cloud but the problem is that this is based on the store listings which can be incorrect. Example of a search of games that doesn't include Steam Cloud: https://steamdb.info/instantsearch/?refinementList%5Bcategories%5D%5B0%5D=-Steam%20Cloud
The problem here is that a lot of the games listed here actually support Steam Cloud, you can tell this on the `Cloud saves` tab, example: https://steamdb.info/app/400/ufs/

So basically I want to search games on SteamDB using the inclusion or omission of the Steam Cloud feature based on the data that's already there on in the `Cloud Saves` tab and not based on the Steam Store listings. (and then filter based on other things as well after that)

I made an issue about this in the incorrect repo earlier: https://github.com/SteamDatabase/FileDetectionRuleSets/issues/547
